src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -65,11 +81,37 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
     </OakLessonBottomNav>
   );
 
+  const handlePrintableResultsClick = async () => {
+    const attemptData = {
+      lessonData: { slug: lessonSlug, title: lessonTitle },
+      browseData: { subject: subject, yearDescription: yearDescription ?? "" },
+      sectionResults: sectionResults,
+    };
+    const attemptId = await logAttempt(attemptData, true);
+    if (attemptId)
+      window.open(
+        pageType === "canonical"
+          ? resolveOakHref({
+              page: "pupil-lesson-results-canonical",
+              lessonSlug,
+              attemptId,
+            })
+          : resolveOakHref({
+              page: "pupil-lesson-results",
+              programmeSlug,
+              unitSlug,
+              lessonSlug,
+              attemptId,
+            }),
+        "_blank",
+      );
+  };
+
   return (
     <OakLessonLayout
       bottomNavSlot={bottomNavSlot}
       lessonSectionName={"review"}
-      phase={phase}
+      phase={phase as "primary" | "secondary"}
```

Comment:
don't cast . resolve the ts error by throwing an error if the phase isn't primary or secondary